### ISSUE TITLE
increase output image resolution 

### ISSUE THREAD

**Issue by victorzhucompass** [facebookresearch/synsin#16]
the 256x256 output looks blurry, wondering in the training/inference pipeline, does it support 512x512 or even higher resolution.

**Comment by oawiles** [comment]
yes it does. You can try training with a different --W parameter (say 512). You could even modify the code to support different aspect ratios (but that would be a bit more involved).
